android/src/main/java/com/horcrux/svg/ClipPathView.java

```diff
@@ -32,6 +32,7 @@ void draw(Canvas canvas, Paint paint, float opacity) {
   @Override
   void saveDefinition() {
     getSvgView().defineClipPath(this, mName);
+    invalidate();
   }
```

Comment:
We should probably call something like:
```suggestion
  void saveDefinition() {
    boolean shouldInvalidate = getSvgView().getDefinedClipPath(mName) != null;
    getSvgView().defineClipPath(this, mName);
    if (shouldInvalidate) {
      invalidate();
    }
  }
```
to not invalidate on first render